src/Tools/Tra/paw_tra.f90

```diff
@@ -2349,6 +2351,46 @@ SUBROUTINE WRITEXYZ(NFIL,FRAME,NAT,ID,R)
       RETURN
       END
 !
+!     ...1.........2.........3.........4.........5.........6.........7.........8
+      SUBROUTINE WRITEEXTXYZ(NFIL,FRAME,NAT,ID,R,CELL,TIME)
+!     ******************************************************************
+!     **                                                              **
+!     ******************************************************************
+      USE STRINGS_MODULE
+      INTEGER(4)  ,INTENT(IN) :: NFIL
+      INTEGER(4)  ,INTENT(IN) :: FRAME
+      INTEGER(4)  ,INTENT(IN) :: NAT
+      CHARACTER(2),INTENT(IN) :: ID(NAT)
+      REAL(8)     ,INTENT(IN) :: R(3,NAT)
+      REAL(8)     ,INTENT(IN) :: CELL(9)
+      REAL(8)     ,INTENT(IN) :: TIME
+      INTEGER(4)              :: IAT
+      REAL(8)                 :: ANGSTROM, PICO, SECOND           
+      CHARACTER(100)          :: STRING
+      CHARACTER(200)          :: EXTXYZ
+      CHARACTER(2)            :: SPECIES
+!     ******************************************************************
+      CALL CONSTANTS$GET('ANGSTROM',ANGSTROM)
+      CALL CONSTANTS$GET('PICO', PICO)
+      CALL CONSTANTS$GET('SECOND', SECOND)
+      IF(FRAME.EQ.1) REWIND NFIL
+      WRITE(NFIL,*)NAT
+      WRITE(STRING,FMT='(9F10.5)')CELL/ANGSTROM
+      EXTXYZ='Lattice="'//TRIM(ADJUSTL(STRING))//'" Properties=species:S:1:pos:R:3 Iter='
```

Comment:
Ensure that the code itself is case insensitive. To use strings with explicit uppercase and lowercase sections use the functions "+" and "-" of strings_module.

Reply:
Solved for paw_strc in commit [fc320c5](https://github.com/cp-paw/cp-paw/pull/8/commits/fc320c52102a0af531ac4185d64958a975ac1c75).
Solved for paw_tra in commit [9636c10](https://github.com/cp-paw/cp-paw/pull/8/commits/9636c10fa4891d3915ff08cb7f40062f1d5805e0).